src/TropicalGeometry/matrix.jl

```diff
@@ -41,3 +41,37 @@ end
 function det(A::Matrix{TropicalSemiringElem{minOrMax}}) where {minOrMax<:Union{typeof(min),typeof(max)}}
     return det(matrix(TropicalSemiring{minOrMax},A))
 end
+
+@doc raw"""
+    is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem}, minOrMax)
+
+Check if a collection of vectors in the tropical torus (given as rows of a matrix `A`) are in tropical general position with respect to the `minOrMax` convention.
+
+# Examples
+```jldoctest
+julia> A = matrix(tropical_semiring(),[1 0;0 1])
+[(1)   (0)]
+[(0)   (1)]
+
+julia> is_tropically_generic(A,min)
+true
+```
+"""
+function is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem}, minOrMax)
+  @req convention(A) == minOrMax "Semiring convention not as declared"
+  function helper(A,C,B)
+    for b in subsets(C,B)
+      Polymake.tropical.tsgn(A[b,:]) == 0 && return false
+    end
+    return true
+  end
```

Comment:
Instead of adding this `helper` function and calling `Polymake.tropical.tsgn` on each subset, would it be simpler to just rely on the `minors` command?  Granted, `minors` uses `det`, which is currently not using the fast `Polymake.tropical.tsgn`.  However, I believe Michael has a pull-request open that is about to fix it.

Reply:
Wouldn't calling minors automatically compute **all** minors and we'd need to see if any tropically vanish? The helper function terminates as soon as we find one with tsgn = 0.